### ISSUE TITLE
Confusing INFO messages

### ISSUE THREAD

**Issue by AndyHee** [pacstall/pacstall#967]
### Describe the bug

I'm on the stable (master) branch, but get messages for the development branch.

### To reproduce

Run `pacstall -U`

and get the following output:
```

[+] INFO: Updated from 4.0.1 (pacstall master) -> 4.0.1 (pacstall master)
[+] INFO: You have updated to a development branch.
[*] WARN: Please remember that bugs may arise, and that this branch may not be as stable as master.
```

### Expected behavior

To get messages relevant to the current branch and not messages that contradict one another.

### Screenshots

N/A

### OS version

Linux/ GNU Devuan 5.0

### Pacstall version

4.0.1, master

**Comment by Zahrun** [comment]
Same here

**Comment by Elsie19** [comment]
Fixed in #976.

**Comment by AndyHee** [comment]
Great, thanks!
